docker-compose.yml

```diff
@@ -99,8 +99,13 @@ services:
           +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
           3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
           -----END RSA PRIVATE KEY-----
-      - NHAIS_MESH_CYPHER_TO_MAILBOX=XX11=nhais_mailbox
-          TES5=nhais_mailbox
+      - NHAIS_MESH_CYPHER_TO_MAILBOX=XX11=responder_mailbox
```

Comment:
I'd stick with nhais_mailbox here - more clear that this is the mailbox we use to send messages to NHAIS